Felienne/hedy
# Fix failing translation of InvalidTypeCombination exception #2449

**Please fill out this template by replacing all content between < >**

< Fix failing translation of InvalidTypeCombination exception >

**Description**

< Fix failing translation of InvalidTypeCombination exception > Use present tense without a subject to describe your PR, for example: "Adds translations of levels 1 to 12 to Polish"

**Fixes < #2449 >**

< Always link the number of the issue or of the discussion that your pr concerns. >
Notable exception: adding content may be done without an accompanying issue 

**How to test**

Run Hedy locally, go to level 12 and execute the following:
```
print 5 + 'test'
```
Check that in stead of the generic error about unexpected code, you receive the following error message:
_You cannot use 5 and test with + because one is a number and the other is text. Try changing 5 to text or test to a number._

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
